api/ctx.md

```diff
@@ -89,6 +89,29 @@ app.Get("/user/*", func(c *fiber.Ctx) error {
 ```
 {% endcode %}
 
+## ParamsParser
```

Comment:
Sorting is alphabetical, please move the block to the right place

Reply:
I am modified doc sort by alphabetical,please accept merge